extensions/sql-database-projects/package.nls.json

```diff
@@ -36,5 +36,6 @@
 	"sqlDatabaseProjects.netCoreDoNotAsk": "Whether to prompt the user to install .NET Core when not detected.",
 	"sqlDatabaseProjects.welcome": "No database projects currently open.\n[New Project](command:sqlDatabaseProjects.new)\n[Open Project](command:sqlDatabaseProjects.open)\n[Create Project From Database](command:sqlDatabaseProjects.importDatabase)",
 
-	"sqlDatabaseProjects.insertSqlInputBinding":"Insert SQL Binding into Azure Function"
+	"sqlDatabaseProjects.insertSqlInputBinding":"Add SQL Binding",
+	"azureFunctions.displayName": "Azure Functions"
```

Comment:
Do we really want to have this be grouped with Azure Functions? I guess I don't know the actual best practice - but it seems a little odd to have stuff from one extension show up under the category for another extension. At the very least it might be good to check with the AF team to make sure they're ok with this.

Reply:
hmm that's true. Ok I'll remove it for now and follow up with the AF team on this